content/hi/docs/reference/glossary/application-developer.md

```diff
@@ -0,0 +1,17 @@
+---
+title: एप्लिकेशन डेवलपर (Application Developer)
+id: application-developer
+date: 2018-04-12
+full_link:
+short_description: >
+  एक व्यक्ति जो कुबेरनेट्स क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
+aka:
+tags:
+  - user-type
+---
+
+एक व्यक्ति जो कुबेरनेट्स क्लस्टर में चलने वाले एप्लिकेशन लिखता है।
+
+<!--more-->
+
+एप्लिकेशन डेवलपर्स एप्लिकेशन के एक हिस्से पर ध्यान केंद्रित करते हैं। उनकी कार्य सीमा के आकार में एक उल्लेखनीय अंतर हो सकता है।
```

Comment:
Whar you've suggested is work streams that is significantly different than focus areas. You can accept the suggestion below OR we can work together to figure out a better way to phrase the sentence. WDYT?
```suggestion
एप्लिकेशन डेवलपर्स एप्लिकेशन के एक हिस्से पर ध्यान केंद्रित करते हैं। उनके फोकस का पैमाना आकारस्वरूप काफी भिन्न हो सकता है।
```